### DISCUSSION TITLE
How to enable autoreload in Odoo during live debugging with Doodba?

### DISCUSSION THREAD

**Discussion by gerardvilajuansmc** [Tecnativa/doodba#631] [Q&A]
Hello!

I’m using Doodba and performing live debugging in Odoo with the "Start Odoo and debug Python" option. However, when I make a change in a file, the modifications in the file are not reflected/updated unless I restart the entire process.

I've noticed that without debugging, I get the message:

`odoo.service.server: autoreload: python code updated, autoreload activated`

Is there a way to enable the autoreload feature while live debugging, so I don't have to restart the process every time I make a change?

Thanks in advance!

**Comment by pedrobaeza** [comment]
That's not possible AFAIK
